Exposed an argument to update an existing stack instead of creating a new one

bjmarfito opened the pull request:
- Edited stackSentinel.py to expose the updateStack to enable updating of existing stack instead of creating a new one or editing line no. 428.

bjmarfito committed:
Created an argument to update an existing stack instead of creating a new one

bjmarfito committed:
Update stackSentinel.py

hfattahi commented on contrib/stack/topsStack/stackSentinel.py:
```diff
@@ -183,6 +183,9 @@ def createParser():
     parser.add_argument('-rmFilter', '--rmFilter', dest='rmFilter', action='store_true', default=False,
                         help='Make an extra unwrap file in which filtering effect is removed')
+    
+    parser.add_argument('-U', '--updateStack', dest='updateStack', action='store_true', default=False,
```
Hi @bjmarfito . I think it might be ok to expose this parameter and force updating. But I don't see where your exposed parameter gets used?

bjmarfito replied:
Thank you @hfattahi for the review. You are correct, I forgot to include the line of code so it can be used during processing. I will add it ASAP.

bjmarfito committed:
Changes relate to updateStack argument

I have changed updateStack codes to inps.updateStack to be able to use the updateStack argument.

bjmarfito committed:
Update stackSentinel.py

bjmarfito committed:
Update stackSentinel.py
